monitor/api/src/main/java/org/datacleaner/monitor/configuration/TenantContext.java

```diff
@@ -162,4 +163,6 @@
      * @return
      */
     public ComponentStore getComponentStore();
+
+    public StringConverter getStringConverter();
```

Comment:
This is not a nice thing to add to this interface. It's a very low-level thing to request from a very high-level interface.